spec.html

```diff
@@ -12767,18 +12791,25 @@ <h1>Runtime Semantics: PropertyDefinitionEvaluation</h1>
         <emu-alg>
```

Comment:
Arguably, `[oldids="sec-__proto__-property-names-in-object-initializers"]` should be added to the `<emu‑grammar>` element preceding this line, instead of being added to “**Static Semantics: Early Errors**” of “**Object Initializer**”.